drivers/sensor/nrf5/temp_nrf5.c

```diff
@@ -82,11 +84,10 @@ static int temp_nrf5_channel_get(struct device *dev,
 
 static void temp_nrf5_isr(void *arg)
 {
-	volatile NRF_TEMP_Type *temp = NRF_TEMP;
 	struct device *dev = (struct device *)arg;
 	struct temp_nrf5_data *data = dev->driver_data;
 
-	temp->EVENTS_DATARDY = 0;
+	nrf_temp_addr->EVENTS_DATARDY = 0;
```

Comment:
Not using the nRFx HAL nrf_temp.h since this looks a bit outdated. Let me know if this is something we should be doing.